UNIXFS.md

```diff
@@ -90,7 +96,9 @@ UnixFS currently supports two optional metadata fields:
   - The remaining 20 bits are reserved for future use, and are subject to change. Spec implementations **MUST** handle bits they do not expect as follows:
     - For future-proofing the (de)serialization layer must preserve the entire uint32 value during clone/copy operations, modifying only bit values that have a well defined meaning: `clonedValue = ( modifiedBits & 07777 ) | ( originalValue & 0xFFFFF000 )`
     - Implementations of this spec must proactively mask off bits without a defined meaning in the implemented version of the spec: `interpretedValue = originalValue & 07777`
-* `mtime` -- The modification time in seconds since the epoch. This defaults to the unix epoch if unspecified
+* `mtime` -- A two-element structure ( `EpochSeconds`, `EpochNanoseconds` ) representing the modification time in seconds relative to the unix epoch `1970-01-01T00:00:00Z`. In contexts where an mtime is mandatory ( e.g. FUSE interfaces ) implementations must treat an unspecified mtime as `0`.
```

Comment:
Minor suggestion, but do we have to include “Epoch” in these names?

Reply:
Also, EpochNanoseconds is _not_ the number of nanoseconds since the unix epoch as the name may suggest

Reply:
Maybe something like this?

```
message EpochTime {
	required int64 Seconds = 1;
	optional fixed32 NanosecondFragment = 2;
}
```

Reply:
+1 - I'd be ok with this suggestion.

Reply:
Hm... in order to be explicit which `Epoch` are we talking about, and to not sound dorky, how about:
```
message UnixTime {
	required int64 Seconds = 1;
	optional fixed32 FractionalNanoSeconds = 2;
}
```

?

Reply:
LGTM

Reply:
@alanshaw can I get a +1 on the naming proposed at https://github.com/ipfs/specs/pull/236#discussion_r363317171 so we can augment/close this PR?

Reply:
haha, much better 🚢🚢🚢

(I was debating whether it was worth bringing up the specific epoch issue but I had decided I it didn't matter too much 🤣)